src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -30,45 +29,60 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
+    public String accountValidation(Account account) throws ServiceException {
```

Comment:
Wondering if instead of returning a String it would be better to have a custom `ValidationResponse` object.
This might contain for example some errorCode and message. We can return all validation issues encountered or we can go only with the first one found.